### PR TITLE
Simplify updateVersion.sh to not use templates

### PR DESCRIPTION
**Pull request by frankbu**
This changes `updateVersion.sh` to directly update istio.VERSION and the installation yaml files (`istio.yaml` and `istio-auth.yaml`), without needing the `install/kubernetes/templates` files.

Next, we should make a similar change in kubeTest.sh and then we can cleanup and remove the whole templates directory.

### CODE REVIEW

**Comment by sebastienvas** [comment]
Since both e2e framework are using the templates, that means that new changes to the istio files will not be tested, so I d rather have the e2e changed at the same time.

**Comment by andraxylia** [comment]
This change is not good nor needed until we unify istio.yaml and istio-auth.yaml, because it now forces the same update in two files (istio.yaml and istio-auth.yaml).
It is also harder to maintain a large file.
We need the ability to mix and match.

**Comment by frankbu** [comment]
@andraxylia: managing two files (the two that are actually used), which will eventually be merged into one is a lot easier than managing 10 files (8 template files that generate two more).

@sebastienvas: I agree with you. I updated kubeTest.sh to not use templates. Where is the other framework that needs to be changed?

**Comment by andraxylia** [comment]
What I meant relates to the case when we need to make a change related to manager or mixer. Now, we only need to change one file, istio-manager.yaml, or istio-mixer.yam, and run updateVersions.sh.

With your proposal, we need to edit two files, istio.yaml and istio-auth.yaml, and run updateVersions.sh.

It is also better to have granularity in the files that need maintaining, manager separate from mixer, egress, ingress, and the ability to mix and match various permutations of those files.

**Comment by sebastienvas** [comment]
To @andraxylia's point, we had base template and other files were generated. Now we actually need to go over the generated files to update them and do it multiple times. 

Let's wait for after the release cut to simplify all that.

**Comment by frankbu** [comment]
Updated kubernetes.go: PTAL @sebastienvas 

I think this simplification is needed. We don't need to generate anything (just update the images). If we want to break it up into multiple files, we should do that, but that's a separate discussion vs templates or not.

Let's discuss at scrum call.

**Comment by frankbu** [comment]
The presubmit is failing and it looks like it's because I haven't got the BUILD file(s) configured right. I'm in over my head with bazel :-(

@sebastienvas: can you please take a look at the error and let me know what I've got wrong?

File: tests/e2e/framework/BUILD [hunk 1]
```diff
@@ -11,7 +11,7 @@ go_library(
     ],
     data = [
         "//:istio_version",
-        "//install/kubernetes/templates",
+        "//install/kubernetes",
```
Comment (sebastienvas) [inline tests/e2e/framework/BUILD]: you called the filegroup templates so you need to change this to:
//install/kubernetes:templates

I understand where the confusion comes from:
//install/kubernetes/templates:templates can be simplified to //install/kubernetes/templates

**Comment by rshriram** [comment]
I agree with frank on this. Look at PR #227 for instance.. We are updating 4 files with the same bit. Two templates and two main..

**Comment by frankbu** [comment]
Close this in favor of #229

**Pull request closed without merging**